recipe/meta.yaml

```diff
@@ -15,20 +16,20 @@ build:
 
 requirements:
   host:
-    - python >=3.8
+    - python >=3.9
     - pip
   run:
-    - casadi
+    - python >=3.9
+    - ocp 7.7.2
+    - vtk =*=qt*
```

Comment:
is it possible to run without vtk/qt? if so, this should likely be in a separate package in the `outputs` section